apps/SageAccountWeb/src/AppInitializer.tsx

```diff
@@ -36,34 +34,13 @@ function AppInitializer(props: { children?: React.ReactNode }) {
     }
   }, [])
 
-  useEffect(() => {
```

Comment:
Remove query parameter redirectURL, and use appURL in SourceApp configuration found in the source table. https://www.synapse.org/#!Synapse:syn45291362/tables/